src/main/java/com/teachmeskills/employee_info/util/EmployeeInfo.java

```diff
@@ -0,0 +1,101 @@
+package com.teachmeskills.employee_info.util;
+
+import com.teachmeskills.employee_info.model.Employee;
+import org.springframework.stereotype.Service;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+@Service
+public class EmployeeInfo {
+    public static Employee getInfo(int id) throws SQLException {
```

Comment:
The should be no static methods in service

Reply:
Fixed.